modules/standard/GMP.chpl

```diff
@@ -51,7 +183,8 @@ module GMP {
      routines if not for ref.
    */
 
-  // initializing rng
+  // Initializing random state 
+  /* */
```

Comment:
Are you done documenting this section?  These extern routines are a little difficult to read at the moment, and there's a couple with a "chpl_" prefix that you may want to "no doc".